asteroid/mad.py

```diff
@@ -284,9 +284,9 @@ def _handle_help(self,_):
       print("print <name>[@<num>|<name>]+|* [-v]\t\t- print contents of <name>, * lists all vars in scope, recursively access (nested) objects with @, '-v' enables verbose printing of nested data")
       print("quit\t\t\t- quit debugger")
       print("set [<func>|<line#> [<file>]]\n\t\t\t- set a breakpoint")
-      print("stack\t\t\t- display runtime stack")
+      print("stack [<num>|* [-v]]\t\t\t- display runtime stack, list all items in specific frame with an index or all frames with '*', '-v' toggles verbose printing")
       print("step\t\t\t- step to next executable statement")
-      print("trace\t\t\t- display runtime stack")
+      print("trace [<num>|* [-v]]\t\t\t- display runtime stack, list all items in specific frame with an index or all frames with '*', '-v' toggles verbose printing")
```

Comment:
Hmm, trace is supposed to simply display the runtime stack with no additional information, that how it is different from stack.  One thing that we might talk about is that trace will accept a range of stack frame numbers.  But it should never prints contents of frames.

Reply:
Okay, that makes sense. I assumed both `stack` and `trace` would implement the feature since both of those commands are tied to the same function in the lookup table. I can change that so `trace` only prints the runtime stack.

As for a range of stack frame numbers, the idea I had in mind would be similar to `range()` in Python, where `trace 5` prints the first 5 stack frames, but `trace 1 5` prints the frames from frame 1 up to 5. Does this sound good or was there a different idea you had in mind?